zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -134,7 +138,7 @@ lazy_static! {
 ///
 /// [7.10.1]: https://zips.z.cash/protocol/protocol.pdf#zip214fundingstreams
 pub const FUNDING_STREAM_ADDRESS_CHANGE_INTERVAL: Height =
-    Height(POST_BLOSSOM_HALVING_INTERVAL.0 / 48);
+    Height(POST_BLOSSOM_HALVING_INTERVAL as u32 / 48);
```

Comment:
Optional: This seems like it's a HeightDiff, but we can make that change in another PR if you want.

Reply:
155ce35d1d + 3238412dca